delivery.yaml

```diff
@@ -14,10 +14,14 @@ pipeline:
       fi
       export IMAGE
       make deps shortcheck
-      cd packaging && make docker-build docker-push
+      git status
+      git diff
```

Comment:
shall we keep the diff in the build?

Reply:
let's merge now and see if this really fixed it for the merge to master case :)